_pages/utility-classes.md

```diff
@@ -184,75 +184,97 @@ someOtherObject = CORE.getPersistence().getUser().getAttributes().get("someKey")
 
 ## DocumentQuery
 
-*DocumentQuery* extends *ProjectionQuery* and provides the ability to
-retrieve persisted beans in a type-safe and secure way, without building SQL or OQL Strings.
+*DocumentQuery* extends *ProjectionQuery* and provides a type-safe and secure way to retrieve persisted beans without requiring developers to write SQL or OQL strings.
 
-Key benefits of the DocumentQuery approach are:
-1. to abstract the developer from implementation-specific SQL - so that applications can remain database independent, 
-2. to allow compile-time checks on query code, and
-3. to ensure a consistent security and performance via Skyve's Persistence singleton object.
+Key Benefits:
+1. Abstracts developers from database-specific SQL, ensuring database independence.
+2. Enables compile-time checks on query code for better reliability.
+3. Ensures consistent security and performance via Skyve’s Persistence singleton.
 
-A trivial example of DocumentQuery is to retrieve typed beans using a simple filter:
+### Simple Example
+
+Retrieve a list of inactive users using a basic filter:
 
 ```java
-// retrieve a list of inactive users
+// Retrieve a list of inactive users
 DocumentQuery q = CORE.getPersistence().newDocumentQuery(User.MODULE_NAME, User.DOCUMENT_NAME);
 q.getFilter().addEquals(User.inactivePropertyName, Boolean.TRUE);
 
 List<UserExtension> inactiveUsers = q.beanResults();
-for(UserExtension u : inactiveUsers) {
-  ...
+for (UserExtension u : inactiveUsers) {
+  // Process each user
 }
 ```
 
-### Complex Example 
+### Using OR Filters
```

Comment:
Can we add an intermediate example before the OR filter and complex example? E.g. chaining more than one filter together to produce an AND query?